app/persona/api/authentication/AuthenticationServiceImpl.scala

```diff
@@ -0,0 +1,38 @@
+package persona.api.authentication
+
+
+import java.util.UUID
+
+import com.google.inject.Inject
+import com.mohiva.play.silhouette.api.LoginInfo
+import com.mohiva.play.silhouette.impl.providers.CommonSocialProfile
+import persona.model.authentication.User
+import persona.model.dao.authentication.UserDAO
+
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
As per previous comment, consider import play.api.libs.concurrent.Execution.Implicits._.  Read the documentation here: https://www.playframework.com/documentation/2.4.x/ThreadPools.  Decide if the work you're doing is blocking or non-blocking.  If it's non-blocking, switch to the play thread pool.  If it's blocking, switch to the play thread pool and open a ticket reminding us to do more investigation in the future